common/src/main/java/com/wynntils/features/inventory/ImprovedInventorySyncFeature.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © Wynntils 2021-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.inventory;
+
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.persisted.Persisted;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.Config;
+import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.mc.event.ContainerClickEvent;
+import com.wynntils.utils.mc.McUtils;
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+import java.util.Arrays;
+import net.minecraft.network.protocol.game.ServerboundContainerClickPacket;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.item.ItemStack;
+import net.neoforged.bus.api.EventPriority;
+import net.neoforged.bus.api.SubscribeEvent;
+
+@ConfigCategory(Category.INVENTORY)
+public class ImprovedInventorySyncFeature extends Feature {
+    @Persisted
+    public Config<Boolean> forceSync = new Config<>(false);
```

Comment:
You can have this as `true` as default. I don't doubt that it works and is helpful, if you claim it is. All I wanted was a way to disable it. If you leave this as false as default, not many players will turn it on.

Reply:
But it delays inventory actions? Parts of the fix are enabled by default, and this more "hardcore" one is not. Am I correct in this?

Reply:
We're talking about two things. The forceSync config is for controlling the re-sync packets from the InventoryHandler. That is what I propose to be on by default.

The Feature itself is, I believe, the extra safety-belts sychronization thing that probably is not needed iirc. That should be off by default. I can't see right now if it is or not (github cuts just before that) but if it isn't, it should be.

Reply:
Ah, I see now that `isImprovedSyncEnabled` checks if this feature is enabled; I thought it checked if forceSync was turned on. Then I agree, it's all fine!